### PR TITLE
Create Lib/gftools/push-stats.py

### PR DESCRIPTION
**Pull request by davelab6**
@m4rc1e better this be here than a gist

### CODE REVIEW

**Comment by m4rc1e** [comment]
Thanks Dave! I kept it as a gist because its functionality is limited and it's not prod worthy. If you want, we can improve this and add more features then merge?

**Comment by m4rc1e** [comment]
I enjoyed the fact you added my name to the docstring 🤣

File: Lib/gftools/push-stats.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""Report how many fonts have been added/updated between two dates
+
+Usage: python push-stats.py from_date to_date
+       python push-stats.py 2020-01-01 2020-06-01
+
+WARNING: was written in < 30 mins by Marc Foley. Do not expect miracles :)
```
Comment (chrissimpkins) [inline Lib/gftools/push-stats.py:6]: Lol

**Comment by m4rc1e** [comment]
I'll start upgrading this script since there's interest for it.

**Comment by m4rc1e** [comment]
Let's merge this. We can always improve it later on.

**Pull request merged**